plugin/rewrite/rewrite.go

```diff
@@ -31,37 +31,29 @@ const (
 
 // Rewrite is a plugin to rewrite requests internally before being handled.
 type Rewrite struct {
-	Next     plugin.Handler
-	Rules    []Rule
-	noRevert bool
+	Next      plugin.Handler
+	Rules     []Rule
+	noRevert  bool // for test
```

Comment:
could you add a TODO here, that we want to get rid of these 'used for testing members' in this struct.

Reply:
I could propose a general `RevertPolicy` instead, that can officially be set using the API.
Please give me a trigger to push that proposal. So far, I just added the TODO.